map_merge/src/map_merge.cpp

```diff
@@ -60,11 +60,14 @@ MapMerge::MapMerge() : subscriptions_size_(0)
                                 robot_map_updates_topic_, "map_updates");
   private_nh.param<std::string>("robot_namespace", robot_namespace_, "");
   private_nh.param<std::string>("merged_map_topic", merged_map_topic, "map");
-  private_nh.param<std::string>("world_frame", world_frame_, "world");
+  private_nh.param<std::string>("world_frame", world_frame_, "map");
```

Comment:
Yes, but it is because in my opinion each robot map should have the namespace of the robot, and then, the global map, just be called map. 
But it is just how I imagine it. I can revert this change before the pull request.